api/db/drop.js

```diff
@@ -5,6 +5,14 @@ const { connectDb, closeDb } = require('./connection');
 
 const Interest = require('../models/Interest');
 const User = require('../models/User');
+const Event = require('../models/Event');
+const Attendant = require('../models/Attendant');
+const { createPrompt } = require('./utils');
```

Comment:
This should probably go on line 5 (and leaving a blank line before the models' imports).